libgoal/libgoal.go

```diff
@@ -935,78 +935,81 @@ func MakeDryrunState(client Client, txnOrStxn interface{}, other []transactions.
 	return v2.DryrunRequestFromGenerated(&gdr)
 }
 
-// MakeDryrunStateGenerated function creates DryrunRequest data structure and serializes it into a file
+// MakeDryrunStateGenerated function creates generatedV2.DryrunRequest data structure and serializes it into a file
```

Comment:
this comment confuses me because I don't see file-io happening, I see a datastructure returned